### ISSUE TITLE
Using keras built-in models

### ISSUE THREAD

**Issue by elbaro** [tensorflow/tensorflow#12202]
#### Version
v1.2.0-rc2-21-g12f033d

#### Problem
The Keras built-in models in `tf.contrib.keras.applications.*` cannot be used as a subgraph in TF.

#### Example
https://stackoverflow.com/questions/45585546/error-with-tf-contrib-keras-tf-placeholder

#### Cause
Calling `keras.applications.InceptionV3(weights='imagenet')(input_tensor)` is supposed to load pre-trained weights only for related variables, but it initializes the entire TF graph.

**Comment by reedwm** [comment]
@fchollet can you take a look?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @fchollet: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by fchollet** [comment]
That's correct, all graph variables are initialized in the Keras session when you run Keras models (or load weights in Keras models), that's intended behavior. Keras simply couldn't work without this behavior (it's impossible to rely on users to explicitly initialize variables that they create themselves before using them together with Keras models).
